Extension/src/LanguageServer/client.ts

```diff
@@ -1046,8 +1051,12 @@ export class DefaultClient implements Client {
                     settings.defaultCompiler = compilerDefaults.compilerPath;
                     compilerDefaults = await this.requestCompiler(compilerPaths);
                     DefaultClient.updateClientConfigurations();
+                    action = "configure compiler";
+                    telemetry.logLanguageServerEvent('compilerNotification', { action });
                 } else if (value === selectCompiler) {
                     this.handleCompilerQuickPick(true);
+                    action = "show quickpick";
+                    telemetry.logLanguageServerEvent('compilerNotification', { action });
                 } else {
                     this.showPrompt(selectCompiler, true);
```

Comment:
I think we'd want to log this one too.  "dismissed" would be an appropriate name.